### DISCUSSION TITLE
these blueiris settings seem to work, test them out

### DISCUSSION THREAD

**Discussion by vincom** [gtxaspec/wz_mini_hacks#273] [General]
its been about 24 hrs with the new blueiris v5 settings.  it all seems good. testing 3 separate cameras at the same time, all  v3.
each camera has 2 profiles in blue iris, timelapse record and motion detection to trigger record w/no time lapse.
**make sure both  profiles for each cam match video source configuration, bi doesnt behave correctly if same ip camera is configured differently**.

first camera: on stock fw w/mini hack w/ethernet (better kb rate than wifi)
second camera:  on stock fw w/mini hack wifi
third camera: wyze rtsp fw, no mini hack(last 12hrs no internet access)

bi set to rtsp as ip video source
setup to record every 15min interval, time lapse 3fps, record lenght was 2m15s/
every hour of record i might lose a few seconds due to lose of stream, once inawhile ~1min. so for a 24hr period maybe 2min lost, ymmv
it didnt not matter if camera did not have internet access, wyze app could not connect to it, it did not have  any effect in streaming video even with the wyze rtsp fw. ymmv

blueiris config pics shown here, i somehow cant get pics to link to this post correctly
https://forums.wyzecam.com/t/beta-testing-for-wyze-cam-v3-rtsp-firmware-now-available/196922/754?u=vincom

**Comment by virmaior** [comment]
the `&load=fdasdsafdadfsxxfdas` at the end isn't necessary. It avoids caching from browsers.
